### PR TITLE
14602-Show court order document in ledger

### PR DESCRIPTION
**Pull request by ketaki-deodhar**
*Issue #:* /bcgov/entity#14602

*Description of changes:*
Update textarea label for court order

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: tests/unit/AddStaffNotationDialog.spec.ts [hunk 1]
```diff
@@ -360,8 +360,7 @@ describe('AddStaffNotationDialog', () => {
       {
         propsData: {
           dialog: true,
-          displayName: 'Court Order',
-          name: 'courtOrder'
```
Comment (severinbeauvais) [inline tests/unit/AddStaffNotationDialog.spec.ts:364]: You can't remove `name`. First, it's a required prop. Second, it's used to determine the type of filing.

File: src/components/dialogs/AddStaffNotationDialog.vue [hunk 2]
```diff
@@ -211,6 +211,14 @@ export default class AddStaffNotationDialog extends Vue {
     return ''
   }
 
+  get notationLabel (): string {
+    if (this.isCourtOrder) {
+      return `${this.displayName} Text`
+    } else {
+      return `${this.displayName}`
+    }
+  }
```
Comment (severinbeauvais) [inline src/components/dialogs/AddStaffNotationDialog.vue:220]: What is the UX requirement that led to this change. Can it be a common change for all staff notation filings?

File: tests/unit/AddStaffNotationDialog.spec.ts (hunk 1 above)
Comment (ketaki-deodhar) [reply tests/unit/AddStaffNotationDialog.spec.ts:364]: Totally my bad. Should not have been committed like this. Thanks for pointing out.

File: src/components/dialogs/AddStaffNotationDialog.vue [hunk 3]
```diff
@@ -35,7 +35,7 @@
             v-model="notation"
```
Comment (severinbeauvais) [inline src/components/dialogs/AddStaffNotationDialog.vue:35]: Re: line 30

Was there a UX requirement to change "Detail" to "details"?

File: src/components/dialogs/AddStaffNotationDialog.vue (hunk 2 above)
Comment (ketaki-deodhar) [reply src/components/dialogs/AddStaffNotationDialog.vue:220]: image

**Review by ChrisGabel23 — APPROVED** [review]

File: src/components/dialogs/AddStaffNotationDialog.vue (hunk 2 above)
Comment (severinbeauvais) [reply src/components/dialogs/AddStaffNotationDialog.vue:222]: Beautiful 👏

**Review by severinbeauvais — APPROVED** [review]

File: src/components/dialogs/AddStaffNotationDialog.vue (hunk 3 above)
Comment (severinbeauvais) [reply src/components/dialogs/AddStaffNotationDialog.vue:35]: @ketaki-deodhar ^^

Comment (ketaki-deodhar) [reply src/components/dialogs/AddStaffNotationDialog.vue:35]: @severinbeauvais, not in this ticket

**Comment by severinbeauvais** [comment]
LGTM. Merge when ready.

**Pull request merged**